README.md

```diff
@@ -1 +1,1205 @@
+[Build]   [main] Step 2/8 : WORKDIR /usr/src/app
+[Build]   [main]  ---> Running in 446517b1afdb
+[Build]   [main] Removing intermediate container 446517b1afdb
+[Build]   [main]  ---> c27a2f22ba27
+[Build]   [main] Step 3/8 : COPY package.json package-lock.json ./
+[Build]   [main]  ---> fc79948ab18b
+[Build]   [main] Step 4/8 : RUN npm install --ci --production     && npm cache clean --force     && rm -f /tmp/*
+[Build]   [main]  ---> Running in 59c1b1cbe571
+[Build]   [main] added 50 packages from 37 contributors and audited 126 packages in 9.178s
+[Build]   [main] found 0 vulnerabilities
+[Build]   [main] npm
+[Build]   [main]
+[Build]   [main] WARN
+[Build]   [main]
+[Build]   [main] using --force
+[Build]   [main]  I sure hope you know what you are doing.
+[Build]
+[Build]   [main] Removing intermediate container 59c1b1cbe571
+[Build]   [main]  ---> 0cdc6d1d7af9
+[Build]   [main] Step 5/8 : COPY . ./
+[Build]   [main]  ---> 23e41b46ee6f
+[Build]   [main] Step 6/8 : CMD ["npm", "start"]
+[Build]   [main]  ---> Running in c62943c5e22f
+[Build]   [main] Removing intermediate container c62943c5e22f
+[Build]   [main]  ---> 9434eb22cc67
+[Build]   [main] Step 7/8 : LABEL io.resin.local.image=1
+[Build]   [main]  ---> Running in 9023bae27f0f
+[Build]   [main] Removing intermediate container 9023bae27f0f
+[Build]   [main]  ---> 7e2603a523f2
+[Build]   [main] Step 8/8 : LABEL io.resin.local.service=main
+[Build]   [main]  ---> Running in 05feac72072e
+[Build]   [main] Removing intermediate container 05feac72072e
+[Build]   [main]  ---> 9ef9a4510175
+[Build]   [main] Successfully built 9ef9a4510175
+[Build]   [main] Successfully tagged local_image_main:latest
+
+[Info]    Streaming device logs...
+[Live]    Watching for file changes...
+[Live]    Waiting for device state to settle...
+[Logs]    [9/5/2019, 2:34:37 PM] Creating network 'default'
+[Logs]    [9/5/2019, 2:34:37 PM] Creating volume 'resin-data'
+[Logs]    [9/5/2019, 2:34:37 PM] Installing service 'main sha256:9ef9a45101757ee81aa26d5ca43713289b2e99401d1b13e32842523876fde664'
+[Logs]    [9/5/2019, 2:34:38 PM] Installed service 'main sha256:9ef9a45101757ee81aa26d5ca43713289b2e99401d1b13e32842523876fde664'
+[Logs]    [9/5/2019, 2:34:38 PM] Starting service 'main sha256:9ef9a45101757ee81aa26d5ca43713289b2e99401d1b13e32842523876fde664'
+[Logs]    [9/5/2019, 2:34:39 PM] Started service 'main sha256:9ef9a45101757ee81aa26d5ca43713289b2e99401d1b13e32842523876fde664'
+[Live]    Device state settled
+[Logs]    [9/5/2019, 2:34:41 PM] [main]
+[Logs]    [9/5/2019, 2:34:41 PM] [main] > balena-cli-masterclass@1.0.0 start /usr/src/app
+[Logs]    [9/5/2019, 2:34:41 PM] [main] > node src/helloworld.js; sleep infinity
+[Logs]    [9/5/2019, 2:34:41 PM] [main]
+[Logs]    [9/5/2019, 2:34:41 PM] [main] Hello world!
+```
+Once the code has been built on the device, it immediately starts executing
+and logs are output to the console. You can halt the connection to the local
+device by using `Ctrl-C`.
```

Comment:
might be worth mentioning that the containers keep running even after disconnecting

Reply:
Really good point, yes!